code/model/TimedNotice.php

```diff
@@ -178,6 +184,60 @@ public function canCreate($member = null)
     }
 
     /**
+     * Gets any notices relevant to the present time, context and current users
+     *
+     * @return ArrayList
+     **/
+    public static function get_notices()
+    {
+        // analyse the context this message gets requested
+        $context = self::getContext();
```

Comment:
Needs to be renamed, gontContext() doesn't exist. Just wondering though also, would it not be better to add an optional parameter to ```get_notices($context = null)``` so that we can just pass in 'Web' or 'CMS' as appropriate? Not providing a value would mean all notices would be returned.